go.mod1

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/pingcap/badger v1.5.1-0.20200908111422-2e78ee155d19
 	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
 	github.com/pingcap/errors v0.11.5-0.20200729012136-4e113ddee29e
-	github.com/pingcap/kvproto v0.0.0-20200927054727-1290113160f0
+	github.com/pingcap/kvproto master
```

Comment:
Why not use a specific version?

Reply:
Oh, sorry, I ran `make test` before committing and it doesn't copy go.mod back to go.mod1